ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -316,7 +409,43 @@ gcp_rename(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *from, c
 }
 
 static int
-gcp_file_size(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name, wt_off_t *sizep)
+gcp_file_lock(WT_FILE_HANDLE *file_handle, WT_SESSION *session, bool lock)
+{
+    // Locks are always granted.
+    WT_UNUSED(file_handle);
+    WT_UNUSED(session);
+    WT_UNUSED(lock);
+
+    return 0;
+}
+
+static int
+gcp_file_size(WT_FILE_HANDLE *file_handle, WT_SESSION *session, wt_off_t *sizep)
+{
+    WT_UNUSED(session);
+
+    gcp_file_handle *gcp_fh = reinterpret_cast<gcp_file_handle *>(file_handle);
+    std::string object_key = gcp_fh->name;
```

Comment:
Same comment as before, do we need this object_key variable given that it's only used once?

Reply:
addressed here: [9faf10f](https://github.com/wiredtiger/wiredtiger/pull/8777/commits/9faf10fe9b57e6c7c29f5909f2cb8c35c5a3ec8e)